packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -59,6 +59,37 @@ def test_if_marathon_app_can_be_deployed_with_mesos_containerizer(cluster):
     cluster.marathon.deploy_test_app_and_check(app, test_uuid)
 
 
+def test_if_marathon_pods_can_be_deployed_with_mesos_containerizer(cluster):
+    """Marathon pods deployment integration test using the Mesos Containerizer
+
+    This test verifies that a Marathon pods can be deployed.
+    """
+
+    test_uuid = uuid.uuid4().hex
+
+    pod_definition = {
+        'id': '/integration-test-pods-{}'.format(test_uuid),
+        'scaling': {'kind': 'fixed', 'instances': 1},
+        'environment': {'PING': 'PONG'},
+        'containers': [
```

Comment:
I still have the same complaint here and I am upgrading it from a nit. Doing an echo test is a complete false sense of security. Just do
```
exec: {command {shell: touch foo}}
healthcheck: {command: {shell: test -f foo}}
```
Unless we have a health check, we have no idea if the deployment is remotely behaving as expected